### ISSUE TITLE
If credentials file starts with newline it is considered invalid

### ISSUE THREAD

**Issue by tsloughter** [brendanhay/amazonka#290]
I discovered this because CircleCI generates the `~/.aws/credentials` file on the instances and it has a newline at the beginning, like:

```
$ cat .aws/credentials 

[default]
aws_access_key_id = ...
aws_secret_access_key =  ...
```

This leads to the error `InvalidFileError ... key aws_access_key_id Couldn't find section: default`

**Comment by brendanhay** [comment]
Thanks for reporting, I'll fix it shortly.

**Comment by brendanhay** [comment]
Same issue as reported in the `ini` package [here](https://github.com/chrisdone/ini/issues/14).

**Comment by brendanhay** [comment]
The fix has been released as part of `ini-0.3.5`. I'll need to wait for that to appear in the stackage nightly, and then will release a version of `amazonka` with the constrained dependency.

**Comment by tsloughter** [comment]
Great, thanks!
